beanstalk_worker/worker_dockerfile_lint.py

```diff
@@ -137,11 +138,22 @@ def main():
         try:
             job = bs.reserve()
             job_data = json.loads(job.body)
+
+            debug_logs_file = os.path.join(
+                job_data["logs_dir"],
+                "service_debug.log"
```

Comment:
Why are we hard coding when it seems to be same in multiple places. Why not have it in config